presto-oracle/src/main/java/io/prestosql/plugin/oracle/OracleClientModule.java

```diff
@@ -51,10 +51,20 @@ public static ConnectionFactory connectionFactory(BaseJdbcConfig config, Credent
         Properties connectionProperties = new Properties();
         connectionProperties.setProperty(OracleConnection.CONNECTION_PROPERTY_INCLUDE_SYNONYMS, String.valueOf(oracleConfig.isSynonymsEnabled()));
 
-        return new DriverConnectionFactory(
-                new OracleDriver(),
-                config.getConnectionUrl(),
-                connectionProperties,
-                credentialProvider);
+        if (oracleConfig.isConnectionPoolEnabled()) {
+            return new OraclePoolConnectorFactory(
+                    config.getConnectionUrl(),
+                    connectionProperties,
+                    credentialProvider,
+                    oracleConfig.getConnectionPoolMinSize(),
+                    oracleConfig.getConnectionPoolMaxSize());
+        }
+        else {
```

Comment:
Nit: the "else" is redundant since the above always returns

Reply:
done